entity/src/extracted_licensing_infos.rs

```diff
@@ -0,0 +1,32 @@
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel)]
+#[sea_orm(table_name = "extracted_licensing_infos")]
```

Comment:
such a long name.. how about naming it something shorter like `license_info`

Reply:
@bxf12315 it's a fair comment, please rename it.